docs/tests/typo3.bats

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bats
+
+setup() {
+  PROJNAME=my-typo3-site
+  load 'common-setup'
+  _common_setup
+}
+
+# executed after each test
+teardown() {
+  _common_teardown
+}
+
+@test "TYPO3 composer based quickstart with $(ddev --version)" {
+  # mkdir my-typo3-site && cd my-typo3-site
+  run mkdir my-typo3-site && cd my-typo3-site
+  assert_success
+  # ddev config --project-type=typo3 --docroot=public --php-version=8.3
+  run ddev config --project-type=typo3 --docroot=public --php-version=8.3
+  assert_success
+  # ddev start
+  run ddev start
+  assert_success
+  # ddev composer create "typo3/cms-base-distribution"
+  run ddev composer create "typo3/cms-base-distribution"
+  assert_success
+  # ddev exec touch public/FIRST_INSTALL
+  run ddev exec touch public/FIRST_INSTALL
+  assert_success
+  # ddev launch
+  run bash -c "DDEV_DEBUG=true ddev launch"
+  assert_output --partial "FULLURL https://${PROJNAME}.ddev.site"
+  assert_success
+  # validate running project
+  run curl -sfI https://${PROJNAME}.ddev.site
+  assert_success
+  assert_output --partial "location: /typo3/install.php"
+  assert_output --partial "HTTP/2 302"
```

Comment:
As we talked in Discord, it would be wonderful if we could actually validate that the install..php can be used, but that's for another time as we learn how to do this better.